projects/layer-control/src/lib/layer-control/layer-control.component.html

```diff
@@ -1,6 +1,7 @@
 <div cdkDropList (cdkDropListDropped)="drop($event)">
   <!-- (cdkDropListSorted)="sort($event)" -->
-  <div class="list-item" *ngFor="let group of layergroups | reverse;" cdkDragLockAxis="y" cdkDrag>
+  <div class="list-item" *ngFor="let group of layergroups | reverse | itemsfilter: checkClassHide" cdkDragLockAxis="y"
```

Comment:
This will get updated in https://github.com/dlr-eoc/ukis-frontend-libraries/pull/71/commits/31934a08d4429b90a90fa06062b7750357a194d0 because reordering layers did not work with the filter